Dockerfile.slim

```diff
@@ -0,0 +1,42 @@
+#---------------------------------------------------------------------------------------------
+# See LICENSE in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+ARG PYTHON_VERSION="3.11"
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /wheels
+
+RUN pip install --upgrade pip \
+    && pip install wheel
+
+COPY . .
+
+RUN pip wheel .
+
+FROM python:${PYTHON_VERSION}-slim
+
+# See http://label-schema.org for metadata schema
+# TODO: Add `build-date` and `version`
+LABEL maintainer="ApeWorX" \
```

Comment:
Question: Would there be a nice way to share Dockerfile-logic between this one and the other one? I noticed they do some of the same things, and I don't want to update one and forget to update the other one.

Reply:
I think as long as `-slim` was built first, that this one could use `-slim` as it's base to install `recommended-plugins` over

Reply:
@fubuloubu you are correct, I'm just working on the implementation locally to ensure I'm not missing anything

Reply:
I'm having issues keeping this in the python:3.11-slim though, which is the hold up

Reply:
This has been updated. Slim builds with `Dockerfile.slim`, recommended plugins is built in the `Dockerfile`. The logic is tidied up here now.